lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -44,27 +44,27 @@ class DecidirGateway < Gateway
       def initialize(options={})
         requires!(options, :api_key)
         super
-        @options[:two_phase_capture] ||= false
+        @options[:preauth_mode] ||= false
       end
 
       def purchase(money, payment, options={})
-        raise ArgumentError, 'Purchase is not supported for two-phase transactions' if @options[:two_phase_capture]
+        raise ArgumentError, 'Purchase is not supported for two-step transactions' if @options[:preauth_mode]
```

Comment:
Let's also make this a little more descriptive, something like "Purchase is not supported on Decidir gateways configured with the preauth_mode option"?

Reply:
@curiousepic Should I update the message for `auth`, `capture` and `verify`?

Reply:
Yes plz.